### ISSUE TITLE
Preview links with issues/PR from private repos and secret gist

### ISSUE THREAD

**Issue by powerman** [keybase/managed-bots#242]
It should be very useful to implement this feature. I suppose this means bot must run unrestricted to be able to read links in all messages. :disappointed: Maybe there is a way to let bot read only messages which begins with `https://github.com/` and `https://gist.github.com/` substrings?

**Comment by joshblum** [comment]
We could expose a `!github show <url>` command (jirabot does this) or if you just @-mention the bot it can scan the message for URLs to preview. cc @hyperobject

**Comment by powerman** [comment]
@-mention sounds more convenient.

BTW, I'm surprised to get an answer so fast - I was thinking you guys will have no time to work on keybase at all in next several months at least (because of Zoom). It's nice to see keybase isn't abandoned.

**Comment by hyperobject** [comment]
> We could expose a `!github show <url>` command (jirabot does this) or if you just @-mention the bot it can scan the message for URLs to preview. cc @hyperobject

yeah this would probably be pretty doable! would need to make sure that the github app already has the permissions we need to preview links, but i *think* we do?

**Comment by LADYBEAR7819** [comment]
> It would be very useful to implement this feature. I suppose this means bot must run unrestricted to be able to read links in all messages.  Maybe there is a way to let bot read only messages which begins with `https://github.com/` and `https://gist.github.com/` substrings?

**Comment by LADYBEAR7819** [comment]
(https://gist.github.com)
